model/lists/ListService.php

```diff
@@ -82,6 +87,18 @@ public function getListElements(RdfClass $listClass, $sort = true, $limit = 0)
             new ValueCollectionSearchInput($request)
         );
 
+        //todo: this is checking if the list is empty for specific language and fixes the system lists
+        if (
+            !count($result)
+            && !($listClass->isSubClassOf($this->getClass(TaoOntology::CLASS_URI_LIST))
+                && $listClass->getUri() !== tao_models_classes_LanguageService::CLASS_URI_LANGUAGES)
```

Comment:
I would extract this to a private method with an explanatory name, just to help future developers :)